src/features/validationProvider.ts

```diff
@@ -243,14 +243,16 @@ export default class AnsibleValidationProvider {
 			let diagnostics: DiagnosticsDictionary = {};
 			let processLine = (line: string) => {
 				let matches = line.match(AnsibleValidationProvider.matchExpression);
-				this.output.appendLine(`Found:\n${matches}`);
+				this.output.appendLine(`Found: ${matches}\n`);
 				if (matches) {
 					let message = matches.groups?.message ?? "unknown";
-					let line = parseInt(matches.groups?.line ?? "1") - 1;
+					let line = parseInt(matches.groups?.line ?? "0");
```

Comment:
Do you have a test for this?

Reply:
In fact I have no tests and this will soon come to hunt me. If one of you could add even a sample test I could start adding tests. The reason for not adding tests is because I not dot know how to write them for extensions.

Anyway, this morning I proved that there is no bug in the extension and was caused by https://github.com/ansible-community/ansible-lint/pull/1556 -- the linter will be released soon.

Reply:
IIRC the last time I had to setup tests I followed https://code.visualstudio.com/api/working-with-extensions/testing-extension